report-handling-procedure.md

```diff
@@ -0,0 +1,197 @@
+:point_left: [Come back to index](README.md)
+
+# Chicago Python Users Group Code of Conduct
+
+The following guide was borrowed almost-entirely from the excellent Django Code
+of Conduct [Report Handling
+Procedure](https://github.com/django/code-of-conduct/blob/master/reports.md).
+
+## Report handling procedure
+
+This document aims to present the way we handle received reports. This is not a
+complete guide, but a set of guidelines or best practices that guide the way we
+provide support to the community. Each issue is different and requires
+individual consideration on a case-to-case basis.
+
+### How we receive reports
+
+All reports received by us are usually sent to the
+[chipy.conduct@gmail.com](mailto:chipy.conduct@gmail.com) email address
+that is automatically forwarded to the on-call receivers of complaints.
```

Comment:
Are there any guidelines on how incidents would be reported? If the person reporting coc violation does not feel safe about revealing their identify, is that acceptable?

Reply:
I added a paragraph to address this.